docs/versioned-plugins/inputs/beats-v6.3.0.asciidoc

```diff
@@ -0,0 +1,385 @@
+:plugin: beats
+:type: input
+:default_codec: plain
+:plugin-uc: Beats
+:plugin-singular: Beat
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: v6.3.0
+:release_date: 2022-03-21
```

Comment:
Note to self (@karenzone):  Beats changes necessitates change to input-elastic_agent to bring it into alignment.